patacrep/songbook/__main__.py

```diff
@@ -127,7 +127,7 @@ def argument_parser(args):
     return options
 
 
-def main(args):
+def main():
```

Comment:
:+1:

~~~python
def main(args=None):
    if args is None:
      args = sys.argv

    ...

    options = argument_parser(args[1:])

    ...

if __name__ == "__main__":
    main()
~~~

So that it works both with `main()` (to use default `sys.argv` arguments) and `main(['my', 'custom', 'list', 'of', 'arguments'])`.